ui/App.svelte

```diff
@@ -103,6 +104,8 @@
   $: sessionIsUnsealed =
     $store.status === remote.Status.Success &&
     $store.data.status === Status.UnsealedSession;
+
+  customProtocolHandler.register();
```

Comment:
Is there a reason we are doing this explicitly instead of doing it as part of a module side effect?

Reply:
Just a personal preference to have things explicit.

Are you suggesting to wrap the contents of `customProtocolHandler.ts` into a class and then instantiate a singleton of that class upon export?

Would this not lead to an unused import TS warning?

Reply:
> Are you suggesting to wrap the contents of `customProtocolHandler.ts` into a class and then instantiate a singleton of that class upon export?

No, I think this would be overkill. I was suggesting executing the function body of `register` in the `customProtocolHandler` module itself and get rid of the `register` function. But there’s no clean advantage to this so let’s leave it like it is. I was just curious.